library.pl

```diff
@@ -8,8 +8,9 @@
 my %arches = (
 
   # https://github.com/docker-library/official-images/blob/master/library/debian
-  default => 'amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x',
-  buster  => 'amd64, arm32v7, arm64v8, i386',
+  default  => 'amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x',
+  bullseye => 'amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x',
```

Comment:
If you want to get ahead of things, you might want to drop to _just_ the set listed in https://wiki.debian.org/LTS ("i386, amd64, armhf and arm64"), since a future update to the Debian image is going to do so. :eyes:

```suggestion
  bullseye => 'amd64, arm32v7, arm64v8, i386',
```